### ISSUE TITLE
Upgrading from old version and unable to connect to remote chrome

### ISSUE THREAD

**Issue by fcanela** [prerender/prerender#582]
Old versions of prerender allowed us to specify chrome host and port. I am being unable to find a way to do it (seems like `chromeHost` and `chromePort` are no longer working parameters).

Is the package still exposing this feature? I found this line which seems to consume an URL but no idea about how to make it work from `prerender` API:

https://github.com/prerender/prerender/blob/master/lib/browsers/chrome.js#L59

**Comment by thoop** [comment]
Which version had that capability? Looks like it must have been removed. Can you explain a little about your setup? Do you have multiple chrome instances running locally or are you just running Chrome on a different machine?

**Comment by fcanela** [comment]
I have one instance running in another container that may be in the same machine or not. We have done that way because we had stability problems with `chrome` in the past and we preferred to have Rancher checking `chrome` health and ensuring it is alive.

After digging a little bit deeper it looks like we were using a custom version with that feature manually added over version 5.2.2.

I can push a PR so `prerender` accepts a remote chrome host and port so everyone can have this feature and we do not have to maintain our own version. Does this sound fine for the project?

**Comment by thoop** [comment]
Yeah I think that sounds great as long as it defaults to the current behavior and lets you override it if needed. Thanks!
